<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow3</class>
 <widget class="QMainWindow" name="MainWindow3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_hasilnumerik">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>800</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hasil Integrasi Numerik</string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_f0fn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>740</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_galatabs">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>740</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_iterasi">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>800</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>i</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>xi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>f(xi)</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_galatexact">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>680</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Galat Exact</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_galatabs">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>740</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Galat Abslut</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sigmaf1fn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>680</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Î£F1 + Fn-1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_galatexact">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>680</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_hasilnumerik">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>860</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_galatrelatif">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>800</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_f0fn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>740</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F0 + Fn</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_sigmaf1fn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>680</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_galatrelatif">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>800</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Galat Relatif</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I Nyoman Yogi Putra Arthawan (2415101060)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>680</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PROGRAM PERHITUNGAN NEWTON COTES: METODE TRAPESIUM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1001</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I Gede Nyoman Arya Adi Prima Merta (2415101024)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
